java/fury-core/src/main/java/io/fury/serializer/Serializers.java

```diff
@@ -410,17 +414,69 @@ public Double read(MemoryBuffer buffer) {
     }
   }
 
-  public static final class StringBuilderSerializer extends Serializer<StringBuilder> {
-    private final StringSerializer stringSerializer;
+  static Tuple2<Function, Function> builderCache;
+
+  private static synchronized Tuple2<Function, Function> getBuilderFunc() {
+    if (builderCache == null) {
+      Function getValue =
+          (Function) Functions.makeGetterFunction(StringBuilder.class.getSuperclass(), "getValue");
```

Comment:
Can we pass `int.class` to `Functions.makeGetterFunction` to get `ToIntFunction<CharSequence>`, otherwise following exception wil lbe raised:
![image](https://github.com/alipay/fury/assets/12445254/a93e0b1c-3e65-4468-9cb3-0a4157ab9606)